.github/sourcemap-url.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+
+const sourceMapURLPattern = /\/\/# sourceMappingURL=(.*)/;
+const sourceMap = fs
+  .readFileSync("./packages/wrangler/wrangler-dist/cli.js")
```

Comment:
Can you pull this string out into a constant, since it is used multiple times in this file.
(It could even be grabbed from the command line args so that it can be configured in the action...?)

Reply:
> It could even be grabbed from the command line args so that it can be configured in the action...?

I think for this we would need to turn the script into an "Action" and pull in the libraries for that. I could look into it though.